src/app/routes/getInitialData/index.js

```diff
@@ -18,7 +20,47 @@ const baseUrl = onClient()
   ? getBaseUrl(window.location.origin)
   : process.env.SIMORGH_BASE_URL;
 
-const getUrl = pathname => `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
+export const validateRendererEnvironment = environment => {
```

Comment:
I'm questioning the importance of logging this user error 🤔 was it a requirement in the issue?

Reply:
Yeah, I don't think the logging is really essential here. There are plenty of other things we probably should be logging but aren't!

Reply:
Agreed - logging removed!